src/TimeModeling/Modeling/distributed.jl

```diff
@@ -201,10 +201,10 @@ end
 Reduce one level of the reduction tree consisting of `nleaf` futures. Each `leaf`
 reduces into itself `futures[i]`.
 """
-function reduce_level!(futures::Vector{_TFuture}, nleaf::Int)
+function reduce_level!(futures::Vector{_TFuture}, nleaf::Int, m::Int, pids::Array{Int})
     nleaf == 0 && return
     @sync for i = 1:nleaf
-        @async remotecall_fetch(local_reduce!, futures[i].where, futures[i], futures[nleaf+i])
```

Comment:
main problem is here should be `m` not `nleaf`

Reply:
From the names/syntax I hardly think you trked to fix it and understand it but only copy pasted and made up a sentence for it

Reply:
I followed DistributedOperations.jl

Reply:
You just took it and brute force copy paste it. without checking what I changed and how it can realte. Here is proper fix:

https://github.com/slimgroup/JUDI.jl/compare/forfrancis?expand=1